frontend/src/components/Timelinecard.vue

```diff
@@ -0,0 +1,27 @@
+<template lang="pug">
+div
+  v-timeline-item(v-for='dat in datas', :key='dat.text', :color='dat.totalcolor', fill-dot='', v-model='dat.side')
```

Comment:
Change to `timelineDetail` in `timelineDetails`

Reply:
Okay will change the variable names.